src/components/Card.js

```diff
@@ -1,12 +1,10 @@
 import React, {useState, useEffect} from "react"
 import axios from "axios"
-import ImageCard from "./imageCard.js"
-//import moment from "moment";    
+import ImageCard from "./imageCard.js"    
 
 export default function Card(props){
     const [card, setCard] = useState([]);
-   // const [date, setDate]=useState(moment().format("MMM Do YY"));
-//https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY TOO MANY REQUESTS
+     
     useEffect(() => {
          axios  
         .get(`https://api.nasa.gov/planetary/apod?api_key=zlqB9lUQl36nWqt1XGT211C1EaSx154A3bZVbvUW`)
```

Comment:
Ana this is great just FYI api_keys added to Github you may get emails regarding this. Reason being is that any API key that is showing in the code is considered a security issue or breach. Because API Keys are linked to someone account or business information and or data.